docs/source/app-dev/explicit-contract-disclosure.rst

```diff
@@ -0,0 +1,150 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _explicit-contract-disclosure:
+
+Explicit contract disclosure (experimental)
+###########################################
+
+Starting with Canton 2.6, explicit contract disclosure is introduced as an (experimental) feature
+that allows seamlessly delegating contract read rights to a non-stakeholder using off-ledger data distribution.
+
+.. note:: Usage of disclosed contracts in command submission by means of explicit contract disclosure
+  can be enabled by toggling the ``explicit-disclosure-unsafe`` flag in the participant configuration as exemplified below.
+  However, the feature is **experimental** and **must** not be used in production environments.
+
+::
+
+    participants {
+        participant {
+            ledger-api.explicit-disclosure-unsafe = true
+        }
+    }
+
+Contract read delegation
+------------------------
+
+Contract read delegation allows a party to acquire read rights during
+command submission over a contract of which it is neither a stakeholder nor an informee.
+
+As an example application where read delegation could be used,
+let's consider a simplified swap transaction between two parties.
+In this example, each party has a football NFT card, modeled below as ``FootballCardNft``,
+that they want to swap.
+
+::
+
+    template ``FootballCardNft``
+      with
+        owner: Party
+        issuer: Party
+        playerReference: Text
+      where
+        signatory [owner, issuer]
+        key (issuer, playerReference): (Party, Text)
+        maintainer key._1
```

Comment:
```suggestion
```

The example does not seem to make use of it ==> minimize; and reap the extra benefit of making keys less prominent, as their support in multi-domain settings is tenuous.